src/richie/apps/persons/tests/test_person_plugin.py

```diff
@@ -81,7 +81,10 @@ def test_person_plugin_render(self):
         # pylint: disable=no-member
         self.assertContains(response, image.file.name)
         # Short resume should be present
-        self.assertContains(response, "A short resume")
+        self.assertInHTML(
+            "<p>A short resume</p>",
+            str(response.content, encoding='utf8'),
+        )
```

Comment:
self.assertContains(..., html=true) looks better because you can pass it directly the response.

Is it because assertInHTML fails and not assertContains when the content is escaped?

Reply:
`assertInHTML` seems more relevant to me because it can not fail because of spaces or indentation around content or inside tag. By the way finally the assertContain may do the same thing internally with response.